app/src/main/java/com/asfoundation/wallet/home/HomeFragment.kt

```diff
@@ -89,6 +89,7 @@ class HomeFragment : BasePageViewFragment(),
     views.sendButton.setOnClickListener { viewModel.onSendClick() }
     views.receiveButton.setOnClickListener { viewModel.onReceiveClick() }
     views.emptyClickableView.setOnClickListener { viewModel.onBalanceClick() }
+    views.currencySelector!!.setOnClickListener { viewModel.onCurrencySelectorClick() }
```

Comment:
The reason why this is nullable and you had to put "!!", is because you forgot to add the currencySelector to landscape layout. (Also why it crashes). Try to avoid "!!" wherever you can, unless there's good reason to.